apps/src/lib/ui/PopUpMenu.jsx

```diff
@@ -0,0 +1,109 @@
+import React, {Component, PropTypes, Children} from 'react';
+import Radium from 'radium';
+import color from '../../util/color';
+
+const TAIL_WIDTH = 14;
+const TAIL_HEIGHT = 12;
+const BACKGROUND_COLOR = color.white;
+const BORDER_COLOR = color.light_gray;
+
+const style = {
+  menu: {
+    position: 'absolute',
+    zIndex: 20,
+    border: `1px solid ${BORDER_COLOR}`,
+    backgroundColor: BACKGROUND_COLOR,
+    borderRadius: 2,
+    boxShadow: "3px 3px 3px gray",
+    marginTop: TAIL_HEIGHT,
+    textAlign: 'center',
+  },
+};
+const tailBorderStyle = {
+  position: 'absolute',
+  bottom: '100%',
+  left: '50%',
+  marginLeft: -TAIL_WIDTH / 2,
+  borderTopWidth: 0,
+  borderBottomWidth: TAIL_HEIGHT,
+  borderLeftWidth: TAIL_WIDTH / 2,
+  borderRightWidth: TAIL_WIDTH / 2,
+  borderStyle: 'solid',
+  borderColor: `transparent transparent ${BORDER_COLOR} transparent`,
+};
+const tailFillStyle = {
+  ...tailBorderStyle,
+  bottom: 'calc(100% - 2px)',
+  borderColor: `transparent transparent ${BACKGROUND_COLOR} transparent`,
+};
+
+class PopUpMenu_ extends Component {
```

Comment:
For this and `Item`, I'm not sure I'd name them with the underscore. They are already local scoped on account of not being exported. I tend to view underscored names as indications that even though you can access this thing "publicy" you aren't intended to do so.